docs/setup/linux.md

```diff
@@ -11,6 +11,12 @@ MetaDescription: Get Visual Studio Code up and running on Linux.
 
 ## Installation
 
+### Ubuntu and [snap supported distros](https://snapcraft.io/docs/core/install) 
+
+Install the snap on [snap supported](https://snapcraft.io/docs/core/install) systems from the store.
+
+    snap install vscode --beta --classic
```

Comment:
We would need to have the package named `code` as all our documentation references `code as the CLI, not `vscode`.